packages/kit/src/core/sync/utils.js

```diff
@@ -29,6 +29,7 @@ export function write(file, code) {
 const dedent_map = new WeakMap();
 
 /**
+ * Removes leading whitespace from each line of the input string
```

Comment:
If I understand correctly now, it doesn't remove all leading whitespace, but just some of it
```suggestion
 * Allows indenting template strings without the extra indentation ending up in the result.
```